Dockerfile-debian.template

```diff
@@ -22,6 +22,10 @@ RUN set -eux; \
 		gsfonts \
 		imagemagick \
 	; \
+# https://github.com/docker-library/ruby/issues/344
+	apt-get download shared-mime-info; \
+	dpkg-deb --fsys-tarfile shared-mime-info_*.deb | tar -C /usr/local -x --strip 2 ./usr/share/mime/packages/freedesktop.org.xml; \
+	rm shared-mime-info_*.deb; \
```

Comment:
Rather than making it extra complex, we should just install the package; the image already has any dependencies:

```console
$ docker run -it --rm redmine bash
Unable to find image 'redmine:latest' locally
latest: Pulling from library/redmine
6f28985ad184: Already exists
f6e107a7a87f: Pull complete
fc649c5e4067: Pull complete
bf3509f3d9aa: Pull complete
d81cb91e5871: Pull complete
1530eeb95029: Pull complete
5228d4e99e13: Pull complete
6afe9dbbfbe3: Pull complete
9b2105c54489: Pull complete
f8b20bdc7b33: Pull complete
7793db22d886: Pull complete
8bcb39d80c04: Pull complete
27ac76d9ac9b: Pull complete
Digest: sha256:c0a05292768084e6199f01dafa3f917a8a62946b6062b449020f749b48d97a7c
Status: Downloaded newer image for redmine:latest
root@e76328677577:/usr/src/redmine# apt-get update
Get:1 http://security.debian.org/debian-security buster/updates InRelease [65.4 kB]
Get:2 http://deb.debian.org/debian buster InRelease [121 kB]
Get:3 http://deb.debian.org/debian buster-updates InRelease [51.9 kB]
Get:4 http://security.debian.org/debian-security buster/updates/main amd64 Packages [269 kB]
Get:5 http://deb.debian.org/debian buster/main amd64 Packages [7907 kB]
Get:6 http://deb.debian.org/debian buster-updates/main amd64 Packages [9504 B]
Fetched 8424 kB in 1s (8665 kB/s)
Reading package lists... Done
root@e76328677577:/usr/src/redmine# apt-get install shared-mime-info
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following NEW packages will be installed:
  shared-mime-info
0 upgraded, 1 newly installed, 0 to remove and 14 not upgraded.
Need to get 766 kB of archives.
After this operation, 4947 kB of additional disk space will be used.
Get:1 http://deb.debian.org/debian buster/main amd64 shared-mime-info amd64 1.10-1 [766 kB]
Fetched 766 kB in 0s (0 B/s)
debconf: delaying package configuration, since apt-utils is not installed
Selecting previously unselected package shared-mime-info.
(Reading database ... 16403 files and directories currently installed.)
Preparing to unpack .../shared-mime-info_1.10-1_amd64.deb ...
Unpacking shared-mime-info (1.10-1) ...
Setting up shared-mime-info (1.10-1) ...
root@e76328677577:/usr/src/redmine# ls -lh /usr/share/mime/packages/freedesktop.org.xml
-rw-r--r-- 1 root root 2.2M Sep 25  2018 /usr/share/mime/packages/freedesktop.org.xml
```

Reply:
sure :)